java/common/common-task/src/test/java/sleeper/task/common/RunCompactionTasksTest.java

```diff
@@ -153,6 +153,45 @@ void shouldCreateTasksToMaxWhenJobsOnQueuePlusExistingTasksGreaterThanMax() {
             assertThat(scaleToHostsRequests).containsExactly(3);
             assertThat(launchTasksRequests).containsExactly(1);
         }
+
+        @Test
+        void shouldCreateNoTasksAndScaleWhenExistingTasksPresentPlusExistingTasksLessThanMax() {
```

Comment:
It's a bit confusing that the other tests use "Plus" to talk about two numbers added together, whereas the new tests just use it as a synonym for "And". It also seems important that there are no jobs on the queue in the new tests.

This could be something like "shouldScaleHostsWhenNoJobsOnQueueAndExistingTasksPresentAndLessThanMax"?